pypeit/scripts/collate_1d.py

```diff
@@ -563,6 +636,7 @@ def get_parser(cls, width=None):
         parser.add_argument('--par_outfile', default=None, type=str,
                             help='Output to save the parameters')
         parser.add_argument('--outdir', type=str, help=blank_par.descr['outdir'] + " Defaults to the current directory.")
+        parser.add_argument('--spec1d_outdir', type=str, help=blank_par.descr['spec1d_outdir'] + " Defaults to overwriting existing spec1ds.")
         parser.add_argument('--tolerance', type=str, help=blank_par.descr['tolerance'])
         parser.add_argument('--match', type=str, choices=blank_par.options['match_using'],
                             help=blank_par.descr['match_using'])
```

Comment:
I have noticed that this parameter has a different name here (`match`) and as paypeitpar (`match_using`). It would be easier if you choose only one of these two for both.

Reply:
I changed it to --match_using. Originally I made them different because I wanted the command line arguments to be shorter and easier to type than the config file versions. But in this case I was able to update this to use --match_using and argparse still accepts --match